reference/settings.v3.yml

```diff
@@ -1404,27 +1449,190 @@ components:
+          example: 1
+        id:
+          type: integer
+          description: Web Analytic ID.
+          example: 1
+        name:
+          type: string
+          description: Web Analytic title.
+          example: Google Analytics
+        tracking_code:
           type: string
+          description: Google Analytic tracking code. Required when version is 1
+          example: GA-1234567890
+        property_id:
+          type: string
+          description: Google Analytic property ID. Required when version is 2
+          example: GA-1234567890
         enabled:
-          description: Flag indicates if Web Analytics Provider is enabled or not.
           type: boolean
+          description: Flag indicates if Web Analytic is enabled or not.
+        data_tag_enabled:
+          type: boolean
+          description: Flag indicates if data tags are enabled by Google Analytic.
+        version:
+          type: integer
+          description: Google Analytic version.
+          example: 1
+    AnalyticVisualWebsiteOptimizer:
+      type: object
+      title: Visual Website Optimizer Analytic
+      properties:
+        channel_id:
+          type: integer
+          description: 'Channel ID. If empty, the default Channel ID will be used.'
+          example: 1
         id:
-          description: Web Analytics Provider ID.
           type: integer
+          description: Web Analytic ID.
+          example: 2
         name:
-          description: Web Analytics Provider title.
           type: string
-      x-internal: false
-    AnalyticsProviders:
-      type: array
-      items:
-        $ref: '#/components/schemas/AnalyticsProvider'
-      x-internal: false
+          description: Web Analytic title.
+          example: Visual Website Optimizer
+        vwo_smartcode:
+          type: string
+          description: Visual Website Optimizer Smartcode.
+          example: VWO-1234567890
+        enabled:
+          type: boolean
+          description: Flag indicates if Web Analytic is enabled or not.
+    AnalyticFacebookPixel:
+      type: object
+      title: Facebook Pixel Analytic
+      properties:
+        channel_id:
+          type: integer
+          description: 'Channel ID. If empty, the default Channel ID will be used.'
+          example: 1
+        id:
+          type: integer
+          description: Web Analytic ID.
+          example: 3
+        name:
+          type: string
+          description: Web Analytic title.
+          example: Facebook Pixel
+        pixel_id:
+          type: string
+          description: Facebook Pixel ID.
+          example: FP-1234567890
+        enabled:
+          type: boolean
+          description: Flag indicates if Web Analytic is enabled or not.
+        is_oauth_connected:
+          type: boolean
+          description: Flag indicates if specific integration service grant OAuth token for Bigcommerce Store
```

Comment:
description: Flag indicates if a specific integration service grants an OAuth token for a Bigcommerce Store.